PublicGitArchive/pga-create/cmd/pga-create/main.go

```diff
@@ -51,3 +52,8 @@ func main() {
 		panic(err)
 	}
 }
+
+func fail(operation string, err error) {
```

Comment:
I would expect this function to be used somewhere? Maybe instead of `panic(err)`?